template/src/pages/signature-mint/ContractDemo.tsx

```diff
@@ -138,8 +138,8 @@ export default function SignatureMintDemo() {
             {sigFailure && (
               <p className="mb-4 rounded-lg bg-yellow-100 p-4 text-sm text-yellow-700" role="alert">
                 <span className="font-medium">Warning!</span> There was a problem with the
-                signature. Please follow the steps below and ensure SIGNATURE_MINT_SIGNER is set in
-                the environment file.
+                signature. Please follow the steps below and ensure SIGNATURE_MINT_PRIVATE_KEY is
```

Comment:
Shoul update the `.env.example` file as well and also the `contracts/README.md` file :)

Reply:
This is only valid for the FE.. We need the private key to sign on the FE and the address to validate within the contract.

Reply:
So in contract we need SIGNATURE_MINT_SIGNER

In the FE we need SIGNATURE_MINT_PRIVATE_KEY

Reply:
Arent all variables/env variables on frontend public? Wouldnt this be a security flaw then

Reply:
Arent all variables/env variables on frontend public? Wouldnt this be a security flaw then